src/query/functions/tests/it/scalars/geo_h3.rs

```diff
@@ -90,3 +94,58 @@ fn test_h3_k_ring(file: &mut impl Write) {
         ("k", UInt32Type::from_data(vec![1, 2, 3, 4])),
     ]);
 }
+
+fn test_h3_is_valid(file: &mut impl Write) {
+    run_ast(file, "h3_is_valid(0)", &[]);
+    run_ast(file, "h3_is_valid(644325524701193974)", &[]);
+
+    run_ast(file, "h3_is_valid(res)", &[(
+        "res",
+        UInt64Type::from_data(vec![
+            1,
+            644325524701193974,
+            644325529094369568,
+            644325528627451570,
+            644325528491955313,
+        ]),
+    )]);
+}
+
+fn test_h3_get_resolution(file: &mut impl Write) {
+    run_ast(file, "h3_get_resolution(0)", &[]);
+    run_ast(file, "h3_get_resolution(644325524701193974)", &[]);
+
+    run_ast(file, "h3_get_resolution(h3)", &[(
+        "h3",
+        UInt64Type::from_data(vec![
+            644325524701193974,
+            644325529094369568,
+            644325528627451570,
+            644325528491955313,
+        ]),
+    )]);
+}
+
+fn test_h3_edge_length_m(file: &mut impl Write) {
+    run_ast(file, "h3_edge_length_m(0)", &[]);
+    run_ast(file, "h3_edge_length_m(1)", &[]);
+    run_ast(file, "h3_edge_length_m(15)", &[]);
+    run_ast(file, "h3_edge_length_m(16)", &[]);
+
+    run_ast(file, "h3_edge_length_m(res)", &[(
+        "res",
+        UInt64Type::from_data(vec![1, 2, 3, 4]),
```

Comment:
Should be UInt8Type since the function arg is UInt8Type.

Reply:
fixed